root-common/usr/share/container-scripts/mysql/init/50-passwd-change.sh

```diff
@@ -4,9 +4,14 @@ password_change() {
   # Set the password for MySQL user and root everytime this container is started.
   # This allows to change the password by editing the deployment configuration.
   if [[ -v MYSQL_USER && -v MYSQL_PASSWORD ]]; then
+    local user_maches=$(echo "SELECT COUNT(*) AS found FROM mysql.user WHERE user='${MYSQL_USER}' AND Host='%' \G" | mysql $mysql_flags)
```

Comment:
Any reason, Why `HOST` is needed? I guess, we are checking only `user`

Reply:
Why host is important: we change the password for `mysql@%`, where `%` is `host`. So, even if there would be e.g. `mysql@192.168.0.0`, it does not help us, because the user would not be found either.